dna/cointoss/cointoss.js

```diff
@@ -1,75 +1,200 @@
+
+
+function anchor(anchorType, anchorText) {
+  return call('anchors', 'anchor', {
+    anchorType: anchorType,
+    anchorText: anchorText
+  }).replace(/"/g, '');
+}
+
+
+function anchorExists(anchorType, anchorText) {
+  return call('anchors', 'exists', {
+    anchorType: anchorType,
+    anchorText: anchorText
+  });
+}
+
 
 // ==============================================================================
 // EXPOSED Functions: visible to the UI, can be called via localhost, web browser, or socket
 // ===============================================================================
 
-var AppID = App.DNA.Hash;
 var Me = App.Key.Hash;
 
 function whoAmI()
 {
     return Me;
 }
 
-// set the handle of this node
-function setHandle(handle) {
 
-    // get old handle (if any)
-    var oldHandle = getAnchor(Me + ":handle");
+// Handles / Anchors
+// ===============================================================================
 
-    // if there was one, remove old handle from directory by index
-    if (oldHandle != null)
-        removeFromListAnchor("userDirectory", undefined, undefined, oldHandle);
+// set the handle of this node
+function setHandle(handle) {
+  debug(
```

Comment:
I think we should remove this mermaid debugging.  I don't think it adds anything to cointoss.